pom.xml

```diff
@@ -88,6 +88,11 @@
             <version>${sonar-maven-plugin.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>uk.gov.companieshouse</groupId>
+            <artifactId>structured-logging</artifactId>
```

Comment:
why do we need structured logging?

Reply:
It's used by the InternalUserInterceptor that we import from api-security-java (and maybe CRUD one as well?) but the api-security-java pom has it as an optional dependency meaning that the importing pom has to supply the dependency
https://github.com/companieshouse/api-security-java/blob/master/pom.xml#L64

Reply:
this seems to me as not how use the optional tag, 
Does importing this here interfere with our current log structure at all?

Reply:
we've noticed that importing structured logging means we lose the application name and class name from the logs. I'll raise a separate jira story to integrate with structured logging.

Reply:
Jira story to use structured-logging - https://companieshouse.atlassian.net/browse/NCS-756